db/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import Column, Integer, String
+
+
+class Base(DeclarativeBase): pass
+
+
+class Person(Base):
+    __tablename__ = "user"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
вот тут индекс сейчас не нужен
```suggestion
    id = Column(Integer, primary_key=True)
```